jupyter_collaboration/handlers.py

```diff
@@ -195,7 +199,9 @@ async def open(self, path):
         self.room = self.websocket_server.get_room(path)
         self.set_file_info(path)
         self.saving_document = None
-        asyncio.create_task(self.websocket_server.serve(self))
+        task = asyncio.create_task(self.websocket_server.serve(self))
+        self.background_tasks.add(task)
```

Comment:
Should we initialize the `background_tasks` attribute in the constructor, or are we using the attribute from the `JupyterWebsocketServer`?

Reply:
Good point, it should be in the `JupyterWebsocketServer`.